k_gains:
  k1: [8.0, 10.3114, 0.0, 0.0]
  k2: [0.0, 0.0, 8.0, 10.3114]

yf: [0.73, 0.0, -0.73, 0.0]
